src/decision/CriteriaWeights.jl

```diff
@@ -1,28 +1,51 @@
-Base.@kwdef struct CriteriaWeights{P,N} <: EcoModel
-    wave_stress::P = Param(
+Base.@kwdef struct CriteriaWeights{P,P2,P3,N} <: EcoModel
+    seed_wave_stress::P = Param(
         1.0;
         ptype="real",
         bounds=(0.0, 1.0),
         default_bounds=(0.0, 1.0),
         dists="unif",
-        name="Wave Stress",
-        description="Importance of avoiding wave stress. Higher values places more weight on areas with low wave stress.",
+        criteria_keywords=(:seed, :weight),
```

Comment:
Could you elaborate on what the `:weight` keyword is used for? The struct is ostensibly for criteria weights, so are we adding a `:weight` keyword for all parameters here?

Reply:
I gather it's to distinguish with `threshold` values - okay leave this as is, we'll clean up later as part of a larger restructure.